### PR TITLE
Little grammar in title

### PR DESCRIPTION
**Pull request by spirillen**
I have broken my tongue for a while with the old title, hoping this one is better

### CODE REVIEW

**Comment by ryanbr** [comment]
Needs better englush here..

**Comment by spirillen** [comment]
> Needs better englush here..

Nicely spotted :wave: 

Made a little change to the "URL(s) for issue" by pre-making the back-ticks ``` not sure if this will make it easier / faster for contributor's but the thought is to make it easier for first time users to understand the meaning of the line, without being a blue genie to English.

I swing both ways... to the question... go wash your thought :rofl:

**Comment by ryanbr** [comment]
`Replace this line with URL(s) for issue`

Not sure what this is trying to include.

**Comment by spirillen** [comment]
The intention:

If you are a first timer to make a report to EL _and_ you do not natively speak English _and maybe_ it is actually your first time writing any issue, then this lines become pretty heard to understand

```
<!-- To prevent tracking, wrap the website URL in a Code tag please. **mandatory** -->
```

For what are `code tags` ?? just take a more reason example from https://github.com/easylist/easylist/issues/9590 (Sorry @bloodyowl it's not to make you stick out or anything, it is just a random picked issue for example case)

<details><summary>Screenshot</summary>

image

</details>

To prevent this, i suggest, made me think, _what if_ we made the back ticks for code blocks and a text for which people what copy / paste the urls('s) over, to avoid misunderstanding, by not understanding or preventing laziness?

```
\`\`\`
Replace this line with URL(s) for issue
\`\`\`
```

Thanks for the feedback and this is the thought behind my suggestion.

**Comment by DandelionSprout** [comment]
This PR makes 3 changes, 2 of which have rookie grammar mistakes I would expect from a 10-year old.

As someone who contracted PTSD after spending *several years* (primarily 2017-19) fixing horrendous translations to Norwegian of apps like VLC, Audacity, XnViewMP, and many more, I sadly feel obliged by moral codes to say: Maybe it's a better idea to not submit all these translation suggestion PRs to various GitHub repos anymore. And especially not for English.

**Comment by spirillen** [comment]
DandelionSprout you got your troll report and blacklisting.  well done :clap:   :cake: :candle:

File: issue_template.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To avoid being banned, don't constantly re-open or create new (related) issue re
 
 <!-- Just include the website URL in the Title line of this issue report -->
 
-### List the website(s) whit issue(s):
+### List the website(s) wiht issue(s):
```
Comment (iam-py-test) [inline issue_template.md:14]: Type - this should be 'with'

File: issue_template.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Replace this line with URL(s) for issue
 
 <!-- Which adblock lists are you're using? -->
 
-### Basic OS and browser informaations
+### Basic OS and browser information's
```
Comment (iam-py-test) [inline issue_template.md:31]: Does this need an _'s_?

Comment (spirillen) [reply issue_template.md:31]: As I recall from my time in school yes, as there are more than one information

But hey, that is what the PR is for, finding errors before pushing.

PS: For your emoji... I'm sorry there isn't a :sob: `:sob:` but still training in my new keyboard and the other new one just like generating doubles. Pretty annoying actually

File: issue_template.md (hunk 1 above)
Comment (iam-py-test) [reply issue_template.md]: Thanks!

**Pull request closed without merging**